roles/kubernetes-apps/registry/defaults/main.yml

```diff
@@ -5,6 +5,18 @@ registry_storage_access_mode: "ReadWriteOnce"
 registry_disk_size: "10Gi"
 registry_port: 5000
 registry_replica_count: 1
+
+# type of service: ClusterIP, LoadBalancer or NodePort
+registry_service_type: "ClusterIP"
+# you can specify your cluster IP address when registry_service_type is ClusterIP
+registry_service_clusterIP: ""
```

Comment:
I have a concern from the mix of underscore and CamelCase string for options.
It is better to define underscore names like registry_service_cluster_ip instead for consistency.